src/styles/toolbar.styl

```diff
@@ -6,6 +6,18 @@
     .coz-btn:last-child
         margin-right 0
 
+.fil-toolbar-trash
+    @extends $buttons
```

Comment:
Should be fixed with the Cozy-UI refactoring as $buttons doesn't exist anymore and the danger-outline is included in the $button placeholder… but until then I don't really have a clean solution :/